openreview/conference/templates/authorWebfield.js

```diff
@@ -19,128 +18,140 @@ var paperDisplayOptions = {
+    location.href = '/login?redirect=' + encodeURIComponent(
+      location.pathname + location.search + location.hash
+    );
+  }
+
   Webfield.ui.setup('#group-container', CONFERENCE_ID);  // required
 
-  Webfield.ui.header(HEADER.title, HEADER.instructions);
-  $('#header').css('margin-bottom', '2rem');
+  renderHeader();
 
-  renderConferenceTabs();
+  OpenBanner.venueHomepageLink(CONFERENCE_ID);
 
   load().then(renderContent).then(Webfield.ui.done);
-
-  OpenBanner.venueHomepageLink(CONFERENCE_ID);
 }
 
 
 // Load makes all the API calls needed to get the data to render the page
-// It returns a jQuery deferred object: https://api.jquery.com/category/deferred-object/
 function load() {
-
-  var authorNotesP;
-  var invitationsP;
-  var edgeInvitationsP;
-
-  if (!user || _.startsWith(user.id, 'guest_')) {
-    authorNotesP = $.Deferred().resolve([]);
-    invitationsP = $.Deferred().resolve([]);
-    edgeInvitationsP = $.Deferred().resolve([]);
-
-  } else {
-    authorNotesP = Webfield.get('/notes', {
-      'content.authorids': user.profile.id,
-      invitation: SUBMISSION_ID,
-      details: 'overwriting'
-    }).then(function(result) {
-      //Get the blind submissions to have backward compatibility with the paper number
-      var originalNotes = result.notes;
-      blindNoteIds = [];
-      originalNotes.forEach(function(note) {
-        if (note.details.overwriting && note.details.overwriting.length) {
-          blindNoteIds.push(note.details.overwriting[0]);
-        }
-      });
-
-      if (blindNoteIds.length) {
-        return Webfield.post('/notes/search', {
-          ids: blindNoteIds
-        })
-        .then(function(result) {
-          return result.notes.filter(function(note) {
-            return note.invitation === BLIND_SUBMISSION_ID;
-          }).map(function(blindNote) {
-            var originalNote = originalNotes.find(function(o) { return o.id == blindNote.original;});
-            blindNote.content.authors = originalNote.content.authors;
-            blindNote.content.authorids = originalNote.content.authorids;
-            return blindNote;
-          });
-        });
-      } else {
-        return result.notes;
+  var authorNotesP = Webfield.get('/notes', {
+    'content.authorids': user.profile.id,
+    invitation: SUBMISSION_ID,
+    details: 'overwriting'
+  }).then(function(result) {
+    //Get the blind submissions to have backward compatibility with the paper number
+    var originalNotes = result.notes;
+    blindNoteIds = [];
+    originalNotes.forEach(function(note) {
+      if (note.details.overwriting && note.details.overwriting.length) {
+        blindNoteIds.push(note.details.overwriting[0]);
       }
     });
 
-    invitationsP = Webfield.get('/invitations', {
-      regex: CONFERENCE_ID + '.*',
-      invitee: true,
-      duedate: true,
-      replyto: true,
-      type: 'notes',
-      details:'replytoNote,repliedNotes'
+    if (blindNoteIds.length) {
```

Comment:
I realized this code is necessary because the only way to know the submissions the user is an author is looking at the original notes but we should render the blinded note.

Reply:
Yes, I was thinking there must be a better way, but the only other way I could think of was getting the note numbers from the Authors groups and using those numbers to query get the blinded notes (with original in details).